conans/test/functional/cmake_test.py

```diff
@@ -526,11 +526,13 @@ def __init__(self, shared=None):
         self.conanfile_directory = "."
         self.source_folder = self.build_folder = "."
         self.settings = None
-        self.options = Options(PackageOptions.loads(""))
         self.deps_cpp_info = namedtuple("deps_cpp_info", "sysroot")("/path/to/sysroot")
         self.output = TestBufferConanOutput()
         if shared is not None:
-            self.options = namedtuple("options", "shared")(shared)
+            self.options = Options(PackageOptions({"shared": ["True", "False"]}))
+            self.options.values = OptionsValues([("shared",  shared)])
```

Comment:
self.options.shared = True

Reply:
It's already assigned because of the default value, right? it's not needed to assign it.